.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+# Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+name: Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
why do we need to specify 0 fetch depth here?

Reply:
It's probably not strictly needed, but it doesn't hurt to fetch the full commit history in the job.